org.flatpak.ClassicSnakeTerminal.yml

```diff
@@ -0,0 +1,50 @@
+app-id: org.flatpak.ClassicSnakeTerminal
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+modules:
+  - name: python
```

Comment:
Could you use the python 2.7 in shared modules? This one here https://github.com/flathub/shared-modules using the same shared build config where possible helps ensure that maximum storage deduplication happens for users.